ddtrace/contrib/celery/signals.py

```diff
@@ -130,3 +130,6 @@ def trace_failure(*args, **kwargs):
         if ex is None:
             return
         span.set_exc_info(ex.type, ex.exception, ex.tb)
+
+def trace_retry(*args, **kwargs):
```

Comment:
From my review, I actually think we have a bug in the `celery` integration by not having this implemented.

It seems that `trace_retry` gets called when a task fails and needs to be retried, so we are not actually capturing/closing the spans on any tasks which need to be retried.

Reply:
I think my statement here is wrong... 

I was having an issue getting traces to show up in the UI when I was doing explicit retries, but from the tests it seems like we are still running `task_postrun` which closes the span and etc.